test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -244,17 +244,17 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
 '@
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
PSModulePath changes interacting with other PowerShell processes.Does not duplicate the System32 module path in subprocesses
```powershell
Expected 1, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 520
520:         $sys32ModPathCount | Should -Be 1
```